### ISSUE TITLE
[Docs] Videos embedding

### ISSUE THREAD

**Issue by calvellido** [arrow-kt/arrow#625]
For a smooth integration for video series into each docs section, we will have a _media_ section at the bottom of the page, not disrupting the direct access to information for an user.

For the user to notice that they can get a full video explanation, an icon/button will be shown at the top, that on click will scroll you to the embedded videos section.

**Comment by calvellido** [comment]
The way we have though it could work best for now is by having a `video` YAML parameter in the markdown files. This param, if set, will be the YouTube video ID, e.g. 
```
---
layout: docs
title: Option
permalink: /docs/datatypes/option/
video: 5SFTbphderE
---
```

And with these settings for an specific doc page, the button/link at the top and the media section at the bottom will be shown, with its proper YouTube video embedded.

**Comment by calvellido** [comment]
Closed by #627 ✌

**Comment by calvellido** [comment]
Closed by #627 ✌
